src/views/pages/Crud.vue

```diff
@@ -167,7 +166,7 @@ function getStatusLabel(status) {
                             <InputIcon>
                                 <i class="pi pi-search" />
                             </InputIcon>
-                            <InputText v-model="filters['global'].value" placeholder="Search..." />
+                            <InputText v-model="filters.global.value" placeholder="Search..." />
```

Comment:
I changed this vs leaving the warning or disabling the warning, but happy to revert it and disable the warning as a 'project norm' override if you like?